code/model/modelrender.cpp

```diff
@@ -863,6 +854,17 @@ void model_render_add_lightning( model_draw_list *scene, model_render_params* in
 			}
 
 			secondary = smi->arc_secondary_color[i];
+      
+      			// try and scale the size a bit so that it looks equally well on smaller vessels
+			width = Arc_width_default_damage;
+			if (pm->rad < Arc_width_no_multiply_over_radius_damage) {
+				width *= (pm->rad * Arc_width_radius_multiplier_damage);
+
+				if (width < Arc_width_minimum_damage) {
+					width = Arc_width_minimum_damage;
+				}
+			}
```

Comment:
You're going to want to copy this width stuff into the MARC_TYPE_DAMAGED case as well now, too.

Reply:
thanks, just pushed